### DISCUSSION TITLE
Stopped receiving updates warning?

### DISCUSSION THREAD

**Discussion by Pinjontall94** [prateekmedia/appimagepool#122] [General]
Is this project EOL? This is honestly one my favorite apps on my linux box so I'd hate to see AppImage Pool go, but flathub has this prominent warning that it hasn't been receiving updates. What's the plan forward?

**Comment by prateekmedia** [comment]
Pushed a runtime update to flathub.

**Reply by Pinjontall94** [reply]
thank you! :)
